Formula/fcct.rb

```diff
@@ -0,0 +1,51 @@
+class Fcct < Formula
+  desc "Fedora CoreOS Config Transpiler"
+  homepage "https://github.com/coreos/fcct"
+  url "https://github.com/coreos/fcct/archive/v0.5.0.tar.gz"
+  sha256 "50912f67ecc2da62b45f597522a685350422998d5840a18828fc9505a5dc51db"
+  head "https://github.com/coreos/fcct.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
+    ENV["GOFLAGS"] = "-mod=vendor"
+    ENV["CGO_ENABLED"] = "0"
+
+    system "go", "build",
+      "-ldflags", "-w -X github.com/coreos/fcct/internal/version.Raw=#{version}",
+      "-o", bin/"fcct", "internal/main.go"
```

Comment:
```suggestion
      *std_go_args, "internal/main.go"
```

This will also insert `-trimpath` which is good for reproducibility.